analysis_options.yaml

```diff
@@ -1,4 +1,13 @@
 include: package:flutter_lints/flutter.yaml
 
-# Additional information about this file can be found at
-# https://dart.dev/guides/language/analysis-options
```

Comment:
What's the principle behind these linting rules? The repo itself was running effective dart linting rules, and if we change it I think it would be best to migrate either to official flutter lints or VGV's linting rules. So I'm curious what standard do these updated rules follow or is it a personal/custom one?

Reply:
Those are just personal linting rules, nothing more.